manifests/cluster-infra/4b_alertmanager_deployment.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: alertmanager-config
+data:
+  alertmanager.yml: |
+    global:
+      resolve_timeout: 5m
+
+    route:
+      group_by: ['alertname', 'namespace']
+      group_wait: 1m
+      group_interval: 5m
+      repeat_interval: 2d
+      receiver: amGithubNotifier-bridge
+
+    receivers:
+    - name: amGithubNotifier-bridge
+      webhook_configs:
+      - send_resolved: false
+        url: 'http://amGithubNotifier/hook'
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: alertmanager
+  labels:
+    app: alertmanager
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: alertmanager
+  template:
+    metadata:
+      labels:
+        app: alertmanager
+    spec:
+      serviceAccountName: prometheus
+      containers:
+      - image: quay.io/prometheus/alertmanager:v0.17.0
+        args:
+        - "--web.external-url=http://{{ .DOMAIN_NAME }}/alertmanager"
+        - "--config.file=/etc/alertmanager/alertmanager.yml"
+        - "--cluster.listen-address="
+        - "--storage.path=/alertmanager"
+        name: alertmanager
+        volumeMounts:
+        - name: config
+          mountPath: /etc/alertmanager/
+        - name: storage
+          mountPath: /alertmanager
+          subPath: alertmanager-data
+        ports:
+        - name: am-web
+          containerPort: 9093
+      volumes:
+      - name: config
+        configMap:
+          name: alertmanager-config
+      - name: storage
+        persistentVolumeClaim:
+          claimName: prometheus-meta
+      terminationGracePeriodSeconds: 300
+      nodeSelector:
+        cloud.google.com/gke-nodepool: prow
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: alertmanager
+  labels:
+    app: alertmanager
+spec:
+  type: NodePort
+  ports:
+  - name: am-web
+    port: 80
+    targetPort: am-web
+  selector:
+    app: alertmanager
+
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
```

Comment:
IIUC this exposes the AlertManager UI and API over the Internet and it could allow people to silence alerts and discard the notifications.

Reply:
should basic auth be added here? or remove the alertmanager UI entirely?

Reply:
Not sure. I'd say that it doesn't need to be exposed for now since it can be accessed by authorized people using port forwarding. @krasi-georgiev thoughts?

Reply:
yep I would say lets disable for now and can think about this in another PR:

https://github.com/kubernetes/ingress-nginx/blob/858e3ff2354fb0f5066a88774b904b2427fb9433/examples/external-auth/nginx/README.md